homeassistant/components/device_tracker/asuswrt.py

```diff
@@ -12,14 +12,37 @@
 from collections import namedtuple
 from datetime import timedelta
 
+import voluptuous as vol
+
 from homeassistant.components.device_tracker import DOMAIN
 from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
-from homeassistant.helpers import validate_config
 from homeassistant.util import Throttle
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.device_tracker import PLATFORM_SCHEMA
```

Comment:
Can be merged with `from homeassistant.components.device_tracker import DOMAIN`.